### PR TITLE
Extend stub to perform schema validation of DES disposal submission.

### PR DESCRIPTION
**Pull request by rszmacinski**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/ReturnController.scala [hunk 1]
```diff
@@ -31,25 +33,43 @@ import uk.gov.hmrc.cgtpropertydisposalsstubs.util.GenUtils.sample
...
 import scala.util.Try
 
 @Singleton
 class ReturnController @Inject() (cc: ControllerComponents) extends BackendController(cc) with Logging {
 
   def submitReturn(cgtReferenceNumber: String): Action[JsValue] =
     Action(parse.json) { request =>
-      val submittedReturn: JsResult[(BigDecimal, LocalDate)] = for {
+
+      import Version7._
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/ReturnController.scala:45]: isn't the version of the schema we have v4?

Comment (rszmacinski) [reply app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/ReturnController.scala:45]: good catch, fixing

Comment (rszmacinski) [reply app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/ReturnController.scala]: fixed

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**